Apps/AccountDataAccess/test/unit/PatientRepositoryTests.cs

```diff
@@ -302,110 +346,125 @@ private static BlockAccessMock SetupBlockAccessMock(bool changeFeedEnabled, bool
                 new PatientQueryFactory(new Mock<IServiceProvider>().Object),
                 messageSender.Object);
 
-            return new(patientRepository, blockedAccessDelegate, messageSender, blockedAccess, audit, commit, command);
+            return new(patientRepository, blockedAccessDelegate, messageSender);
         }
 
-        private static CanAccessDatasourceMock SetupCanAccessDatasourceMock(DataSource dataSource)
+        private static IPatientRepository SetupCanAccessDatasourceMock(HashSet<DataSource> dataSources, bool useCache)
```

Comment:
Method name no longer corresponds to the returned type.  I suggest something like `SetupPatientRepositoryForCanAccessDataSource()`.